src/popup/router/components/InviteItem.vue

```diff
@@ -39,9 +52,12 @@ export default {
   },
   components: { Button, AmountSend, CopyIcon },
   filters: { formatDate },
-  data: () => ({ topUp: false, topUpAmount: 0, balance: 0 }),
+  data: () => ({ topUp: false, topUpAmount: 0, balance: 0, currencySigns }),
   computed: {
-    ...mapState(['sdk', 'current']),
+    ...mapState(['sdk', 'current', 'invites']),
+    tokenBalance() {
```

Comment:
`tokenBalance` is used only in comparisons, can't you just use `balance` instead? I can't get why it is needed to reduce accuracy by `toFixed(2)`